pkg/archive/filesystem.go

```diff
@@ -27,12 +27,16 @@ import (
 )
 
 // GetItemFilePath returns an item's file path once extracted from a Velero backup archive.
-func GetItemFilePath(rootDir, groupResource, namespace, name string) string {
-	switch namespace {
-	case "":
-		return filepath.Join(rootDir, velerov1api.ResourcesDir, groupResource, velerov1api.ClusterScopedDir, name+".json")
-	default:
-		return filepath.Join(rootDir, velerov1api.ResourcesDir, groupResource, velerov1api.NamespaceScopedDir, namespace, name+".json")
+func GetItemFilePath(rootDir, groupResource, namespace, name, versionPath string) string {
```

Comment:
The reason I added it here was it got rid of an if..then block in the calling code, which simplified the complicated backupItem func, since it's completely OK for it to be empty. filepath.Join omits empty strings in the arg, so the if test on empty versionPath is completely unnecessary.

Reply:
But to your point, we could create GetVersionedItemFilePath with the new term added (still OK to be empty), and leave this method signature as is. Actually, that's probably better. I'll do that. So backupItem will call the new versioned one, and the other callers will revert to the older usage.

Reply:
Done. This allowed me to revert the change to restore.go where it calls GetItemFilePath, and now backupItem calls GetVersionedItemFilePath.

Reply:
Thanks!